std/range/package.d

```diff
@@ -9757,10 +9760,10 @@ private struct OnlyResult(T, size_t arity)
         return frontIndex >= backIndex;
     }
 
-    T front() @property
+    CommonType!Values front() @property
     {
         assert(!empty, "Attempting to fetch the front of an empty Only range");
-        return data[frontIndex];
+        return this[0];
```

Comment:
This looks like an error?

Reply:
0 means `frontIndex` in this context, see `opIndex` implementation.

Reply:
Nevermind